Tests/Functional/EqualityWithAccuracy/main.swift

```diff
@@ -0,0 +1,160 @@
+// RUN: %{swiftc} %s -o %T/EqualityWithAccuracy
+// RUN: %T/EqualityWithAccuracy > %t || true
+// RUN: %{xctest_checker} %t %s
+
+#if os(macOS)
+    import SwiftXCTest
+#else
+    import XCTest
+#endif
+
+// Regression test for https://github.com/apple/swift-corelibs-xctest/pull/7
+// and https://github.com/apple/swift-corelibs-xctest/pull/294
```

Comment:
This new test source file merges together the two other test files deleted below, incorporating both of their tests and adding several more examples under the "passing" methods.